Zork Grand Inquisitor: Implement New Game (#2539)
ArchipelagoMW/Archipelago: worlds/zork_grand_inquisitor/docs/setup_en.md

# Zork Grand Inquisitor Randomizer Setup Guide

## Requirements

- Windows OS (Hard required. Client is using memory reading / writing through Win32 API)

> - A copy of Zork Grand Inquisitor. Only the GOG version is supported. The Steam version can work with some tinkering but
>   is not officially supported.

Reviewer: ```suggestion
- A copy of Zork Grand Inquisitor on GOG. The Steam version works with some tinkering but is not officially supported.
```
Nitpicky again, just for briefness

> - ScummVM 2.7.1 64-bit (Important: Will not with any other version. [Direct Download](https://downloads.scummvm.org/frs/scummvm/2.7.1/scummvm-2.7.1-win32-x86_64.zip))

Reviewer: ```suggestion
- ScummVM 2.7.1 64-bit (Important: Will not work with any other version. [Direct Download](https://downloads.scummvm.org/frs/scummvm/2.7.1/scummvm-2.7.1-win32-x86_64.zip))
```
